app/uk/gov/hmrc/economiccrimelevyregistration/services/RegistrationValidationService.scala

```diff
@@ -41,12 +42,16 @@ class RegistrationValidationService @Inject() (clock: Clock, schemaValidator: Sc
   def validateRegistration(registration: Registration): ValidationResult[EclSubscription] =
     transformToEclSubscription(registration) match {
       case Valid(eclSubscription) =>
-        schemaValidator
-          .validateAgainstJsonSchema(
-            eclSubscription.subscription,
-            SchemaLoader.loadSchema("create-ecl-subscription-request.json")
-          )
-          .map(_ => eclSubscription)
+        registration.entityType match {
```

Comment:
It may make more sense to have a separate validation all together for 'Other' type, and lift this pattern match up a layer, that way we will reduce the risk of breaking any existing validation and keep the concerns separate